### ISSUE TITLE
More info in subpackage READMEs (+ example Collections and Items)

### ISSUE THREAD

**Issue by matthewhanson** [stac-utils/stactools#156]
For all stactools subpackages for creation of data, some more detail in the READMEs would be helpful.  

The STAC Extensions README is a good starting point for some of the info to include (e.g., https://github.com/stac-extensions/eo).

Initial thoughts:
- owner email, STAC extensions use a GitHub handle, although since there's no direct messaging in GitHub a contact email is probably better
- short description + links to more info about the dataset.  e.g., in `threedep` I'd like to see a link to a landing page about the dataset where I can learn more about 3DEP data
- List of extensions that this dataset uses
- Table of any new fields this dataset adds that aren't part of an extension (this should be used sparingly but sometimes there's a single field or two to pass through but not worth writing a new extension for, e.g., `threedep:region`)
- Basic STAC Collection(s) for this dataset, linked to from README
- Sample STAC Item(s) for dataset, linked to from README

**Comment by gadomski** [comment]
@matthewhanson I've created https://github.com/stactools-packages/threedep/pull/4 as an example implementation of your requests as outlined above, and I requested your review. The only deviation was on owner email, where I kept it as Github user b/c many (some? most?) Github users have a public email associated with them. Don't feel that strongly about it but I guess it liked it more than an email which can change when people move companies, whatever...happy to use emails too, LMK.
